packages/dev/core/src/Engines/Native/nativeDataStream.ts

```diff
@@ -1,3 +1,4 @@
+import type { DeepImmutable, FloatArray } from "../../types";

```

Comment:
```suggestion
import type { DeepImmutable, FloatArray } from "../../types";

```

Forcing a new build. Can be reverted.